frontend/packages/console-dynamic-plugin-sdk/src/extensions/add-actions.ts

```diff
@@ -7,6 +7,8 @@ export type AddAction = ExtensionDeclaration<
   {
     /** ID used to identify the action. */
     id: string;
+    /** IDs used to identify the action groups the action would belong to. */
+    groupId?: string[];
```

Comment:
Should we want display an action in more than one group? I would expect that `groupId?: string` is enough?

Reply:
yes.
`Event sources` is an example. It appears in `Developer Catalog` as well as `Serverless` groups as per the UXD